spk/fengoffice/Makefile

```
@@ -1,6 +1,6 @@
 SPK_NAME = fengoffice
-SPK_VERS = 3.11.2.8
-SPK_REV = 4
+SPK_VERS = 3.11.4.14
+SPK_REV = 5
 SPK_ICON = src/fengoffice.png
 
 DEPENDS = cross/$(SPK_NAME)
```

avoid makefile variables in DEPENDS for #6255

```suggestion
DEPENDS = cross/fengoffice
```